src/fides/config/__init__.py

```diff
@@ -58,7 +58,7 @@ class FidesConfig(FidesSettings):
         exclude=True,
     )
     oauth_instance: Optional[str] = Field(
-        default=None,
+        default=getenv("FIDES__OAUTH_INSTANCE", None),
```

Comment:
Fixing issue where the `FIDES__OAUTH_INSTANCE` environment variable wasn't available to Fides.